src/EnergyPlus/CurveManager.cc

```diff
@@ -358,33 +352,35 @@ namespace CurveManager {
                                                      ErrorsFound);
             ++CurveNum;
 
+            auto &thisCurve = state.dataCurveManager->PerfCurve(CurveNum);
+
             // could add checks for blank numeric fields, and use field names for errors.
-            state.dataCurveManager->PerfCurve(CurveNum).Name = Alphas(1);
-            state.dataCurveManager->PerfCurve(CurveNum).curveType = CurveType::BiQuadratic;
-            state.dataCurveManager->PerfCurve(CurveNum).ObjectType = CurrentModuleObject;
-            state.dataCurveManager->PerfCurve(CurveNum).NumDims = 2;
-            state.dataCurveManager->PerfCurve(CurveNum).InterpolationType = InterpType::EvaluateCurveToLimits;
-            state.dataCurveManager->PerfCurve(CurveNum).Coeff1 = Numbers(1);
-            state.dataCurveManager->PerfCurve(CurveNum).Coeff2 = Numbers(2);
-            state.dataCurveManager->PerfCurve(CurveNum).Coeff3 = Numbers(3);
-            state.dataCurveManager->PerfCurve(CurveNum).Coeff4 = Numbers(4);
-            state.dataCurveManager->PerfCurve(CurveNum).Coeff5 = Numbers(5);
-            state.dataCurveManager->PerfCurve(CurveNum).Coeff6 = Numbers(6);
-            state.dataCurveManager->PerfCurve(CurveNum).Var1Min = Numbers(7);
-            state.dataCurveManager->PerfCurve(CurveNum).Var1Max = Numbers(8);
-            state.dataCurveManager->PerfCurve(CurveNum).Var2Min = Numbers(9);
-            state.dataCurveManager->PerfCurve(CurveNum).Var2Max = Numbers(10);
+            thisCurve.Name = Alphas(1);
+            thisCurve.curveType = CurveType::BiQuadratic;
+            thisCurve.ObjectType = CurrentModuleObject;
+            thisCurve.NumDims = 2;
+            thisCurve.InterpolationType = InterpType::EvaluateCurveToLimits;
+            thisCurve.coeff[0] = Numbers(1);
+            thisCurve.coeff[1] = Numbers(2);
+            thisCurve.coeff[2] = Numbers(3);
+            thisCurve.coeff[3] = Numbers(4);
+            thisCurve.coeff[4] = Numbers(5);
+            thisCurve.coeff[5] = Numbers(6);
+            thisCurve.inputLimits[0].min = Numbers(7);
```

Comment:
Can do these in a loop too.

Reply:
Definitely could, although I think the index math may add a bit of confusion when mentally processing it and since it's only a few indices, I'm not sure it's worth it here.  As with the other comment, I'm willing to be convinced if you or anyone feels strongly.  I did convert the straight 1-N assignments into loops throughout.